targets/mbed/source/jerry_extapi.cpp

```diff
@@ -86,41 +87,36 @@ static bool
 register_native_function (const char* name,
                           jerry_external_handler_t handler)
 {
-  jerry_api_object_t *global_obj_p;
-  jerry_api_object_t *reg_func_p;
-  jerry_api_value_t reg_value;
-  bool bok;
-
-  global_obj_p = jerry_api_get_global ();
-  reg_func_p = jerry_api_create_external_function (handler);
-
-  if (!(reg_func_p != NULL
-      && jerry_api_is_function (reg_func_p)
-      && jerry_api_is_constructor (reg_func_p)))
+  jerry_value_t global_object_val = jerry_get_global_object ();
+  jerry_value_t reg_function = jerry_create_external_function (handler);
+  jerry_value_t reg_value = jerry_create_undefined ();
+  jerry_value_t set_result;
+
+  if (!(reg_function != 0
```

Comment:
`reg_function != 0` this is not recommended. Don't compare jerry_value_t to a constant integer